corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -130,6 +130,9 @@ hqDefine("geospatial/js/gps_capture",[
             return !self.showLoadingSpinner() && !self.hasError();
         });
 
+        self.createCaseEnabled = ko.observable(false);
```

Comment:
nit: I think a better name would be `creatingCase`. "Enable" seems like its like a toggle or a permission enabled to create case.